src/dev/Web/EastSeat.ResourceIdea.Web/Components/Shared/SidebarNavigation.razor

```diff
@@ -0,0 +1,95 @@
+@* ----------------------------------------------------------------------------------
+    File: SidebarNavigation.razor
+    Path: src/Web/EastSeat.ResourceIdea.Web/Components/Shared/SidebarNavigation.razor
+    Description: Vertical sidebar navigation component with collapsible sections
+   ---------------------------------------------------------------------------------- *@
+
+<div class="sidebar" id="sidebar">
+    <!-- Brand Header -->
+    <div class="sidebar-header">
+        <div class="brand">
+            <div class="brand-logo">
+                <i class="fas fa-circle"></i>
```

Comment:
For the brand logo, resize the image attached here in as the brand logo to be used. I want the logo to be white on a teal background.
![ResourceIdea logo](https://github.com/user-attachments/assets/93c3bcaf-d586-4d6c-90dc-014e13f91b45)